### ISSUE TITLE
API error on signup

### ISSUE THREAD

**Issue by wajeeha-khalid** [dependabot/feedback#134]
I faced API error after i signed up via my github account.

<img width="1159" alt="screen shot 2018-04-20 at 2 18 20 pm" src="https://user-images.githubusercontent.com/6745069/39043042-cec43d10-44a5-11e8-9d85-dece0d80f360.png">

**Comment by wajeeha-khalid** [comment]
Hey Greysteil, I did nothing but sign up via github account and this is the first screen that i saw.
I am using an account which has EdX organization public repo access so may be it picked that up by default.

**Comment by greysteil** [comment]
Ah, that's interesting. On the Dependabot backend I've got a record of the EdX account, but not of your personal one. Did you click through from the GitHub marketplace to sign up, or from the Dependabot website?

(I'm determined to get to the bottom of this - it's been haunting me by very occasionally rearing its head!)

**Comment by wajeeha-khalid** [comment]
Via Dependabot website

<img width="1126" alt="screen shot 2018-04-20 at 2 30 59 pm" src="https://user-images.githubusercontent.com/6745069/39043706-8a660e44-44a7-11e8-946d-00b4f17b24ce.png">

**Comment by greysteil** [comment]
So I'm guessing you saw this screen:

![image](https://user-images.githubusercontent.com/1144873/39043800-3ec5c4c8-4486-11e8-84e2-3fd87038572e.png)

**Comment by greysteil** [comment]
And then this one?

![image](https://user-images.githubusercontent.com/1144873/39043967-b6fe69b8-4486-11e8-88de-59c1576f39e2.png)

**Comment by greysteil** [comment]
And then after that got dumped onto the page that served you an API error?

**Comment by greysteil** [comment]
I can't replicate from my side, which is extremely frustrating.

Can you try visiting https://github.com/apps/dependabot and installing onto your account from there? I think that might fix things.

**Comment by wajeeha-khalid** [comment]
Yes, that was the sequence of steps.
Sure; let me try this URL.

**Comment by wajeeha-khalid** [comment]
yep installing via that app view worked and i am in.
Thanks @greysteil for quickly looking into it and unblocking me.

If you need any help from me for resolving the initial issue; feel free to contact.

**Comment by greysteil** [comment]
Fab, thanks @wajeeha-khalid. I think I just saw an error on our side form one of your Python update runs, so I'll take a look into that 😭! Closing this for now.

**Comment by wajeeha-khalid** [comment]
@greysteil  did you get a chance to look into error for Python update?
I have logged it as a new issue here https://github.com/dependabot/feedback/issues/135
